packages/api/src/hooks/useWithdrawalFiatAddress.ts

```diff
@@ -0,0 +1,25 @@
+import { useCallback, useEffect } from 'react';
+import useMutation from '../useMutation';
+
+const useWithdrawalFiatAddress = (verification_code: string) => {
```

Comment:
Best to consolidate this with: https://github.com/binary-com/deriv-app/pull/10697#pullrequestreview-1707513217 

The hooks in the PR link i shared has a more generic handling of FiatAddress logic

Reply:
Updated